readthedocs/settings/base.py

```diff
@@ -190,8 +185,6 @@ def USE_PROMOS(self):  # noqa
         'django.contrib.auth.middleware.AuthenticationMiddleware',
         'django.contrib.messages.middleware.MessageMiddleware',
         'dj_pagination.middleware.PaginationMiddleware',
-        'readthedocs.core.middleware.SubdomainMiddleware',
-        'readthedocs.core.middleware.SingleVersionMiddleware',
```

Comment:
I think we replace this in the proxito code, so probably need to update that as well.